### PR TITLE
Confirmation screen

### PR DESCRIPTION
**Pull request by anniero98**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR

Added the confirmation screen and styled it as per Figma. Transaction displayed is the latest one from Checkout Screen!

Also added a second testing bypass to get to the confirmation screen (-:

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

### Online sources

[//]: # "Optional - copy links to any tutorial or documentation that was useful to you when working on this PR"

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## How to review
Go through the whole flow and make sure the transaction info matches what's in checkout !

[//]: # "The order in which to review files and what to expect when testing locally"

## Next steps

Navigation gets messed up (i.e when you click the back arrow from CustomerLookupScreen, it goes back to Confirmation Screen), but since we're adding a clerk "logout" button (I think) instead of using the current navigation, it's fine I think

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

## Tests Performed, Edge Cases

[//]: # "Hopefully we will add a testing suite/CI soon, but until then note down the steps you took to test locally"

### Screenshots

image

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @wangannie @thumn

### CODE REVIEW

File: screens/ConfirmationScreen.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            <Body>Points Earned:</Body>
+            <Body>{transaction.pointsEarned}</Body>
+          </SpaceBetweenRowContainer>
+          <SpaceBetweenRowContainer>
+            <Body>Rewards Redeemed:</Body>
+            <Body>{transaction.rewardsApplied}</Body>
+          </SpaceBetweenRowContainer>
+          <SpaceBetweenRowContainer>
+            <Body>Total:</Body>
+            <Body>${transaction.totalPrice}</Body>
```
Comment (wangannie) [inline screens/ConfirmationScreen.js]: Can you add a line that says something like total discounts from rewards that only shows if they've redeemed something?

File: screens/ConfirmationScreen.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+          </SpaceBetweenRowContainer>
+        </ColumnContainer>
+        <FilledButtonContainer onPress={() => this.handleSubmit()} width="236" height="46" style={{ borderRadius: 20 }}>
```
Comment (wangannie) [inline screens/ConfirmationScreen.js]: This button is 40px tall

Comment (wangannie) [reply screens/ConfirmationScreen.js]: Also make this full width of the container (new design updates!)

**Comment by anniero98** [comment]
image
Addressed comments

File: screens/ConfirmationScreen.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    }
+    return (
+      <ColumnContainer style={{ width: '100%', height: '100%', justifyContent: 'center', alignItems: 'center' }}>
+        <ColumnContainer style={{ width: '33%', paddingBottom: 48 }}>
+          <Title>Purchase Recorded! </Title>
+          <Body>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+            dolore magna aliqua. Nec sagittis aliquam malesuada bibendum arcu vitae. Enim praesent elementum facilisis
+            leo. Vulputate mi sit amet mauris commodo quis imperdiet massa. Venenatis a condimentum vitae sapien
+            pellentesque. Id consectetur purus ut faucibus pulvinar elementum.
```
Comment (wangannie) [inline screens/ConfirmationScreen.js]: Actually, can we just get rid of this for now

File: screens/ConfirmationScreen.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+          {transaction.rewardsApplied > 0 && (
+            <SpaceBetweenRowContainer>
+              <Body>Total Discounts:</Body>
+              <Body>${(transaction.rewardsApplied * 5).toFixed(2)}</Body>
+            </SpaceBetweenRowContainer>
+          )}
+          <SpaceBetweenRowContainer>
+            <Body>Total:</Body>
+            <Body>${transaction.totalPrice}</Body>
```
Comment (wangannie) [inline screens/ConfirmationScreen.js]: This doesn't show 2 decimal points (for $5.50, shows 5.5)
